src/interactive.sh

```diff
@@ -0,0 +1,40 @@
+function create_ssh_key()
+{
+  local ssh_dir="$1"
+  local key_name="${2:-kw_ssh_key}"
+
+  if [[ -n "$key_name" ]]; then
+    mkdir -p "$ssh_dir"
+    eval "ssh-keygen -t rsa -f $ssh_dir/$key_name"
```

Comment:
Why do we need `eval`? If we really need to use `eval`, replace it with `cmd_manager`.

Reply:
This was originally intended by us to install ssh at this point. If we are handling it as part of KW's dependencies, then this code is not needed.